src/extensions/surveys/surveys-utils.tsx

```diff
@@ -16,7 +16,7 @@ const window = _window as Window & typeof globalThis
 const document = _document as Document
 const SurveySeenPrefix = 'seenSurvey_'
 
-const logger = createLogger('[Surveys]')
+const logger = createLogger('[Surveys Utils]')
```

Comment:
```suggestion
const logger = createLogger('[Surveys]')
```
any reason to change it? its easier to filter logs by [Surveys] rather than many other options

Reply:
those logs are just not working, i was just testing it out to make sure.

the only logs who are working are from `posthog-surveys`

Reply:
quickfix? :)